### ISSUE TITLE
Embed music or video

### ISSUE THREAD

**Issue by banan314** [jaap-karssenberg/zim-desktop-wiki#506]
It would be nice if we were able to embed a video or a piece of music inside a page so that we can just press play and play it without the need to use a third-party application.

Let me visualize my idea:
![image](https://user-images.githubusercontent.com/17968722/43967039-bea66054-9cc3-11e8-9197-81365c456d9f.png)

**Comment by quesada** [comment]
Agreed. This feature request: "insert video on pages" is more important as time passes. Zim is often used as documentation, and the web has moved towards video more and more. It's becoming very easy to record short videos and insert them on documentation systems.
